sdk/keyvault/Azure.Security.KeyVault.Keys/README.md

```diff
@@ -52,21 +52,21 @@ Use the [Azure CLI][azure_cli] snippet below to create/get client secret credent
         "tenant": "tenant-ID"
     }
     ```
-* Use the returned credentials above to set  **AZURE_CLIENT_ID**(appId), **AZURE_CLIENT_SECRET**(password) and **AZURE_TENANT_ID**(tenant) environment variables. The following example shows a way to do this in Powershell:
+* Use the returned credentials above to set  **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password) and **AZURE_TENANT_ID** (tenantId) environment variables. The following example shows a way to do this in Powershell:
```

Comment:
Need oxford comma.

Reply:
All comments resolved in [PR](https://github.com/Azure/azure-sdk-for-net/pull/11295)